includes/features/onboarding/class-wc-admin-onboarding.php

```diff
@@ -41,28 +41,23 @@ public function __construct() {
 	 * @return bool
 	 */
 	public function should_show_profiler() {
-		// @todo Remove this once we have a proper way to dismiss the profiler.
-		if ( ! defined( 'WOOCOMMERCE_ADMIN_DEV_SHOW_PROFILER' ) || false === WOOCOMMERCE_ADMIN_DEV_SHOW_PROFILER ) {
```

Comment:
This was a short lived change 😆.

Reply:
Haha, I didn't realize it would be gone so quick either.